### DISCUSSION TITLE
Rpc error

### DISCUSSION THREAD

**Discussion by cemalefendi** [Chia-Network/chia-blockchain#8335] [Support]
![Screenshot_20210905-092156](https://user-images.githubusercontent.com/88004149/132117676-841bf7f2-41bf-414c-863d-6d3fec1317c5.jpg)
![Screenshot_20210905-090619](https://user-images.githubusercontent.com/88004149/132117679-ace13bd9-0ede-4260-811b-fc94f57b6de3.jpg)

I see two section for chia in task manager. Is it normal?

And i always give farmer rpc error. How can I solve this problem

I use 1.2.5 version

**Comment by BrandtH22** [comment]
@cemalefendi , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
